packages/demo/test/init.spec.ts

```diff
@@ -50,8 +50,11 @@ describe('Init CLI command', () => {
 
     const ownerAddress = '0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266'
 
-    await execAsync(
-      `npx sphinx init --org-id TEST_ORG_ID --sphinx-api-key TEST_SPHINX_KEY --alchemy-api-key TEST_ALCHEMY_KEY --owner ${ownerAddress}`
+    execSync(
+      `export SPHINX_INTERNAL_TEST__SKIP_GIT=true && npx sphinx init --org-id TEST_ORG_ID --sphinx-api-key TEST_SPHINX_KEY --alchemy-api-key TEST_ALCHEMY_KEY --owner ${ownerAddress}`,
+      {
+        stdio: 'inherit',
+      }
```

Comment:
Fyi, this outputs [logs in our demo tests](https://app.circleci.com/pipelines/github/sphinx-labs/sphinx/5353/workflows/f49ab3c8-b359-4dbb-b76e-404247b58e2c/jobs/26480
) (notice the middle two lines):
>  Init CLI command
> Initializing sample Sphinx project...
> Initialized sample Sphinx project.
>    ✓ Creates and tests a sample Foundry project (41045ms)

Not a big deal, but I don't think we should facilitate this pattern often because it can make the Mocha test cases more difficult to read when there are a lot of tests